src/Core/Rexl.Bind/Operations/BuiltinFunctions.cs

```diff
@@ -378,6 +378,9 @@ private BuiltinFunctions()
         AddOne(TextTrimFunc.TrimStart, new Sig(S.AboutTrimStart, A.Create(S.ArgSource, S.AboutTrimStart_Source)));
         AddOne(TextTrimFunc.TrimEnd, new Sig(S.AboutTrimEnd, A.Create(S.ArgSource, S.AboutTrimEnd_Source)));
 
```

Comment:
We may also want to have just "Pad" or "PadCenter" that pads on both sides to a desired length, keeping the source "centered".